### PR TITLE
Added namespaces to some test fixtures.

### PR DESCRIPTION
**Pull request by SidRoberts**
Hello!

* Type: code quality
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [-] I wrote some tests for this PR
- [-] I have updated the relevant CHANGELOG
- [-] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

I've purposely left the Annotations classes and the Tasks as some of them require no namespace.

### CODE REVIEW

File: tests/_data/fixtures/Di/services.php [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
     ],
     'config' => [
-        'className' => \Phalcon\Config::class,
-        'shared' => true,
+        'className' => Config::class,
+        'shared'    => true,
     ],
     'component' => [
-        'className' => \SomeComponent::class,
+        'className' => SomeComponent::class,
```
Comment (niden) [inline tests/_data/fixtures/Di/services.php]: We needed that to be a single class that has been `required` hence no namespace for this.

File: tests/_data/fixtures/Di/services.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ config:
     shared: true
 
 component:
-    className: \SomeComponent
+    className: \Phalcon\Test\Fixtures\Di\SomeComponent
```
Comment (niden) [inline tests/_data/fixtures/Di/services.yml]: same here

**Review by niden — CHANGES_REQUESTED** [review]

**Comment by SidRoberts** [comment]
I've removed the DI stuff so it should be good to go once the build has completed.

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @SidRoberts

**Pull request merged**